src/dom/epub/epub-view.ts

```diff
@@ -342,6 +346,25 @@ class EPUBView extends DOMView<EPUBViewState, EPUBViewData> {
 		}
 	}
 
+	// Add landmarks with page labels for screen reader navigation
+	private async _addAriaNavigationLandmarks() {
+		for (let [key, value] of this.pageMapping.tree.entries()) {
+			let node = key.startContainer;
+			let textNodeWrapper = closestElement(node);
+			let nextPageElement = textNodeWrapper?.nextElementSibling;
```

Comment:
Oh, the `nextPageElement` was actually just a leftover from my first setup where I was adding the `<nav>` to the first comment AFTER the container of the range. Just because I misunderstood how EPUB tree works a bit. Thanks for pointing that out. 

I cleaned that up and named the wrapper of the range container as `containingElement`.